src/Sulu/Bundle/SnippetBundle/Command/SnippetLocaleCopyCommand.php

```diff
@@ -148,6 +153,8 @@ private function copyDocument($srcLocale, $destLocale, StructureBehavior $docume
             $destLocale,
             Structure::TYPE_SNIPPET
         );
+        $destDocument = $this->documentManager->find($document->getUuid(), $destLocale);
+        $this->documentManager->publish($destDocument, $destLocale);
```

Comment:
Maybe we should add a comment saying this is only necessary as long the snippets do not support publishing by themselves.